integration_tests/rest_api_utils.py

```diff
@@ -15,17 +15,10 @@
 print(f"Integration tests using gateway {BASE_PATH=}")
 DEFAULT_NETWORK_TIMEOUT_SEC = 10
 
-# Generate some fake 24-character user IDs.
-# We don't want different people to get user ID collisions but at the same time we want people to
-# consistently use the same user IDs so that they can clean up their extra endpoints.
-USER_PREFIX = os.getenv("SERVICE_IDENTIFIER", "test")[:8]
-USER_ID_0 = USER_PREFIX + "0" * (24 - len(USER_PREFIX))
-USER_ID_1 = USER_PREFIX + "1" * (24 - len(USER_PREFIX))
-
-DEFAULT_USERS: Sequence[str] = (
-    USER_ID_0,
-    USER_ID_1,
-)
+# Use the scale-launch-integration-tests id
+USER_ID_0 = os.getenv("TEST_USER_ID")  # type: ignore
```

Comment:
set some default value

Reply:
and i think `None` USER_ID_0 caused integration test to fail

Reply:
agreed, I set a default value of "fakeuser"